Apps/phgsgmail/gsgmail_connector.py

```diff
@@ -336,6 +434,75 @@ def _handle_run_query(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_email(self, param):
+
+        # Implement the handler here, some basic code is already in
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Create the credentials with the required scope
+        scopes = ['https://www.googleapis.com/auth/gmail.readonly']
+
+        # Create a service here
```

Comment:
nit: seeing that we have `_create_service` below, maybe this sounds redundant too?

Reply:
are you talking about the comment? if so, see above :P I'm going to take another pass and remove some of these though